packages/hiro-graph-lucene/src/index.js

```diff
@@ -332,24 +338,29 @@ function luceneTerm(context, field, values) {
             `Cannot find '${field}' of type '${context.entity.name}'`
         );
     }
-    return values
-        .map(v => prop.encode(v)) // encode for graphit with our mapping
-        .map(v => (typeof v === "string" ? quote(v) : v)) //quote if needed
-        .map(
-            term =>
-                term === null
-                    ? luceneMissing(context, field) //if term is null, that means the field should be missing.
-                    : `${context.op}${slashForward(prop.src)}:${term}`
-        ) //create querystring
-        .join(" "); //join terms
+    return values.reduce((acc, v) => {
```

Comment:
revert to using multiple `map`s or at least to smthng like `.map(pipedFn).join(' ')`